index.html

```diff
@@ -148,13 +148,13 @@ <h3 class="skills-text">Skills</h3>
         <div class="contact-dev">
           <h2 class="contactmeHeader">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
           <form method="post" autocomplete="off" action="https://formspree.io/f/xdovwjar" class="inputForm">
-            <input id="firstname-input" onchange="saveValue(this);" class="full_Name" type="text" placeholder="Full Name" maxlength="30" name="First-name" required />
-            <div class="fullName">
-              <input id="firstname-input" onchange="saveValue(this);" class="name" type="text" placeholder="First Name" maxlength="30" name="First-name" required />
-              <input id="secondname-input" onchange="saveValue(this);" class="name" type="text" placeholder="Second name" maxlength="30" name="Second-name" required />
-            </div>
+            <input id="fullName" onchange="saveValue(this);" class="full_Name" type="text" placeholder="Full Name" maxlength="30" name="First-name" required />
```

Comment:
Great job team 🤝

- [ ] As stated in the project requirements, If a user submits the contact form with the email field containing uppercase characters an error message should show up. Kindly ensure an error message shows up when an email containing an uppercase letter is used.
- [ ] Also ensure the error message should show up close to the `submit` button or the `input fields`